apps/bfd-server/bfd-server-war/pom.xml

```diff
@@ -298,6 +298,11 @@
 			<artifactId>spark-sketch_2.12</artifactId>
 			<version>2.4.7</version>
 		</dependency>
+		<dependency>
+			<groupId>gov.cms.bfd</groupId>
+			<artifactId>bfd-shared-test-utils</artifactId>
+			<version>${project.version}</version>
```

Comment:
test scope?

Reply:
Since bfd-server uses spring configuration, there is a check in the datasource setup in the SpringConfiguration class that calls database test utils to set up a db in some IT test scenario. I don't immediately know how to tackle fixing that. Might involve making a new spring configuration for the test.

Reply:
Added a comment to the pom about the scoping for this.

Reply:
I think it is worth a TODO and tech debt story to remember the details since this is still not ideal. Thanks for sorting out the other scopes.